src/native/http_library.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include <jni.h>
+#include <crt.h>
+
+#include <aws/common/common.h>
+#include <aws/io/io.h>
+#include <aws/http/connection.h>
+
+
+
+
+JNIEXPORT void JNICALL
+    Java_software_amazon_awssdk_crt_http_HttpLibrary_httpLibraryInit(JNIEnv *env, jclass jni_class) {
```

Comment:
Same as the java class, just move this stuff into the existing crtInit() in crt.c

Reply:
Fixed